scripts/package-env

```diff
@@ -2,9 +2,9 @@
 
 ARCH=${ARCH:-"amd64"}
 SYSTEM_CHART_REPO_DIR=build/system-charts
-SYSTEM_CHART_DEFAULT_BRANCH=${SYSTEM_CHART_DEFAULT_BRANCH:-"dev-v2.10"}
+SYSTEM_CHART_DEFAULT_BRANCH=${SYSTEM_CHART_DEFAULT_BRANCH:-"release-v2.10"}
 CHART_REPO_DIR=build/charts
-CHART_DEFAULT_BRANCH=${CHART_DEFAULT_BRANCH:-"dev-v2.10"}
+CHART_DEFAULT_BRANCH=${CHART_DEFAULT_BRANCH:-"release-v2.10"}
```

Comment:
Doesn't this relate to charts and not KDM? IIUC this value is also taken from the docker file, I think the use of `dev-v2.10` is an intentional fallback